### PR TITLE
Rename bblfsh/dashboard to bblfsh/web

### PR DESCRIPTION
**Pull request by smacker**
Changes for #149 

List of changes in commit messages.

### CODE REVIEW

**Comment by smacker** [comment]
added commit with docs update

**Comment by smacker** [comment]
added comment with imports rename

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The format can be described more formally as follows:
 
 ## Architecture
 
-The Dashboard application consists of 2 parts:
+The web client application consists of 2 parts:
```
Comment (bzz) [inline CONTRIBUTING.md]: Should we use `bblfsh web clinet` for the context instead of just `web client`?

File: src/state/versions.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const initialState = {
   loading: false,
   error: null,
   bblfshdVersion: undefined,
-  dashboardVersion: undefined,
+  webVersion: undefined,
```
Comment (bzz) [inline src/state/versions.js]: may be a `webClientVersion` here?

File: CONTRIBUTING.md (hunk 1 above)
Comment (smacker) [reply CONTRIBUTING.md]: Imo it's unnecessary. If you opened CONTRIBUTING file of the project you most probably know what it does. In this case, you know what it's client for. We also mention babelfish in this document 6 times already. 
Though I don't mind to change to `bblfsh web client`, please confirm once again we really need this change.

File: src/state/versions.js (hunk 2 above)
Comment (smacker) [reply src/state/versions.js]: 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (bzz) [reply CONTRIBUTING.md]: That is reasonable, so to change or not it's up to you. And thanks for double-checking!

Comment (smacker) [reply CONTRIBUTING.md]: I changed it on the first appearance in this document to make it more clear. But below I left it just as `web client`.

**Comment by smacker** [comment]
merge after CI is green.

**Pull request merged**